src/xdo.nim

```diff
@@ -28,7 +28,7 @@ const
     "191":"/","192":"`","219":"[","220":"\\","221":"]","222":"'"
   }.toTable  ## Statically compiled JSON that maps KeyCodes integers Versus Keys strings.
 
-types
+type
```

Comment:
Oh, wow, good catch, I introduced the error, I tracked it back to https://github.com/juancarlospaco/nim-xdo/commit/70872926be831dff32774d6bfa9db8d6be7bdaea#diff-8b7305597ed7f7ceb1592ac2ee2df68d

I dont take too much time on it because pure Enum are deprecated anyways.